cityofaustin/atd-moped
# Enables direct select component editing ✍️

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/13116

## Testing
**URL to test:** [Netlify](https://deploy-preview-1092--atd-moped-main.netlify.app/) or local


**Steps to test:**
1. Start creating a new component by selecting the **Access Control - Driveway Closure**. component type and then **Continue** to enter the map editor.
2. Use the draw button to begin drawing a new line. After you double-click to finish drawing, do not save. Instead, click on the line you just drew, and then click on one of circles that appear at each line vertex. Drag vertex you clicked on. Then click elsewhere on the map to disable editing. Save the component.
3. Edit this component you just created, again dragging vertices. Try deleting a vertex as well. Save your component and and refresh the page to make sure edits are persisted.
4. Create a point-type component (e.g. **Intersection Improvement**) and make sure you can select, draw, and edit the component like normal.


---
#### Ship list
